lizmap/modules/presentation/controllers/service.classic.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * PHP proxy to execute presentation request.
+ *
+ * @author    3liz
+ * @copyright 2017 3liz
+ *
+ * @see      http://3liz.com
+ *
+ * @license    Mozilla Public License
+ */
+class serviceCtrl extends jController
```

Comment:
There is already a bunch of serviceCtrl class into the project (!!). You should use an other name.